pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -137,51 +148,59 @@ func (nim *nodeInfoManager) UninstallCSIDriver(driverName string) error {
 	return nil
 }
 
+func (nim *nodeInfoManager) updateNode(updateFuncs ...nodeUpdateFunc) error {
+	var updateErrs []error
+	err := wait.ExponentialBackoff(updateBackoff, func() (bool, error) {
```

Comment:
Any concern about this blocking for a long time and preventing progress on the caller?

Reply:
The plugin watcher spawns a separate goroutine for each event, so we should be good here